docs/community/README.md

```diff
@@ -1,2 +1,21 @@
-# `jupyter` accessibility community information
+# `jupyter` accessibility community
+
+Once we have assembled the working group, we will list the members here.
+
+## Join our JupyterLab accessibility meetings
+
+Anyone is welcome to attend, if they would like to discuss a topic or just to listen in.
+
+- When: every other Wednesday (check the [community calendar](https://jupyter.readthedocs.io/en/latest/community/content-community.html#jupyter-community-meetings)) at [10:15 AM Pacific Time](https://dateful.com/convert/san-francisco-california?t=1015am)
+- Where: [`jovyan` Zoom](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09)
+- What: discussions on current work and feedback
+
+### Meeting notes and reports
+
+- [current agenda](https://hackmd.io/WnaWXboXSiGoqWvev_fAvA)
+- [public notes archive](https://github.com/jupyterlab/team-compass/issues/98) from past meetings
+<br/><br/>
```

Comment:
Is there a reason to add an HTML tag for a line break here? Maybe I'm just unaware of the reasoning.

```suggestion

```

Reply:
Looking back, I cant recall of any significant reasoning. Most likely just to separate the two bulleted lists for some reason. I am not opposed to combining them though. 

I think maybe my thought was the first bulleted list was more of the "meeting notes" and the second was more of the "reports". Although, the meeting minutes on line 18 would be in the wrong list under that logic.